src/components/app/NavBar/SyncStatus.tsx

```diff
@@ -0,0 +1,57 @@
+import { useSyncStatus } from '../../../hooks';
+import { Network } from '../../../types';
+import { Box } from '@mui/material';
+import Alert from '@mui/material/Alert';
+import Typography from '@mui/material/Typography';
+
+type SyncStatusProps = {
+    network: Network;
+};
+
+export const SyncStatus = (props: SyncStatusProps) => {
+    const { network } = props;
+
+    const {
+        subgraphLoading,
+        subgraphData,
+        subgraphRequestError,
+        rpcLoading,
+        rpcData,
+        rpcRequestError,
+    } = useSyncStatus(network);
+
+    const renderData = () => {
+        if (subgraphLoading || rpcLoading) {
+            return <Box>Loading sync status</Box>;
+        }
+
+        return (
+            <Box>
+                {subgraphRequestError && (
+                    <Alert severity="warning">
+                        Error checking subgraph height
+                    </Alert>
+                )}
+                {rpcRequestError && (
+                    <Alert severity="warning">Error checking RPC height</Alert>
+                )}
+                {!subgraphRequestError &&
+                    !rpcRequestError &&
+                    subgraphData &&
+                    rpcData && (
+                        <Typography variant="caption" align="right">
+                            Subgraph is{' '}
+                            {rpcData.syncHeight
+                                .sub(subgraphData.syncHeight)
+                                .toString()}{' '}
+                            blocks behind
```

Comment:
small nit: I noticed ETH is generally only 1 block behind, so maybe `blocks` should be `block(s)` or we could drop the `s` if the sync height is equal to 1